Makefile

```diff
@@ -36,13 +38,13 @@ clean:
 
 # Main build
 debug:
-	mkdir -p  build/debug && \
-	cmake $(GENERATOR) $(FORCE_COLOR) $(EXTENSION_FLAGS) ${CLIENT_FLAGS} -DEXTENSION_STATIC_BUILD=1 -DCMAKE_BUILD_TYPE=Debug ${BUILD_FLAGS} -S ./duckdb/ -B build/debug && \
+	mkdir -p build/debug && \
```

Comment:
`-DEXTENSION_STATIC_BUILD=1` is already in the ${BUILD_FLAGS} so it's just a clean up